### DISCUSSION TITLE
BGT instruction added

### DISCUSSION THREAD

**Discussion by Peter-Jacob** [adesutherland/CREXX#111] [Progress Updates]
BGT Branch Greater Than   
   BGT label,reg-int-value,reg-max-value
   BGT label,reg-int-value,int-max-value
TestBGT test asm added

This is the one Adrian needs for DO loops

The other ones BGE, BNE, BEQ, BLT, BLE  follow in the next few days

**Comment by adesutherland** [comment]
Many thanks!

**Comment by Peter-Jacob** [comment]
Also added:
BGE Branch if Greater or Equal 
BLT Branch if Less 
BLE Branch if Less or Equal
BEQ Branch if Equal 
BNE Branch if Not Equal

All instructions have label,reg,reg and label,reg,int
The comparisons are on integer only

The compare and branch instructions are complete. All changes have been committed to GIT and contain the appropriate test assembler files
